components/script/dom/element.rs

```diff
@@ -514,6 +514,25 @@ impl Element {
             debug_assert!(false, "Trying to detach a non-attached shadow root");
         }
     }
+
+    // https://html.spec.whatwg.org/multipage/#translation-mode
+    pub fn is_translate_enabled(&self) -> bool {
+        // TODO change this to local_name! when html5ever updates
+        let name = &LocalName::from("translate");
+        if self.has_attribute(name) {
+            match &*self.get_string_attribute(name) {
+                "yes" | "" => return true,
+                "no" => return false,
+                _ => {},
+            }
+        }
+        if let Some(parent) = self.upcast::<Node>().GetParentNode() {
+            if let Some(elem) = parent.downcast::<Element>() {
+                return elem.is_translate_enabled();
+            }
+        }
+        true
```

Comment:
This should be false if this node is not the document element.

Reply:
document.createElement("div").translate is true in Chrome, but it is interesting that the spec doesn't actually say what to do if there's no parent and it's not the document element. https://github.com/whatwg/html/issues/5239